src/main.cpp

```diff
@@ -92,39 +95,44 @@ void loop() {
      * network (WiFiUdp will print an warning message in those cases).
      */
     if (puara.IP1_ready()) { // set namespace and send OSC message for address 1
-        OSCMessage msg1(("/" + puara.get_dmi_name() + "/X").c_str());
-        OSCMessage msg2(("/" + puara.get_dmi_name() + "/Y").c_str());
-        OSCMessage msg3(("/" + puara.get_dmi_name() + "/Z").c_str());
-        msg1.add(event.orientation.x);
-        msg2.add(event.orientation.y);
-        msg3.add(event.orientation.z);
-        Udp.beginPacket(puara.getIP1().c_str(), puara.getPORT1());
-        msg1.send(Udp);
+        
+        // Create bundle to allow sending multiple OSC messages at once
+        OSCBundle bundle;
+
+        String baseMessage = ("/" + puara.dmi_name()).c_str();
+
+        OSCMessage msgOrientation((baseMessage + "/Orientation").c_str());
+        OSCMessage msgAcceleration((baseMessage + "/Acceleration").c_str());
+        OSCMessage msgGyroscope((baseMessage + "/Gyroscope").c_str());
```

Comment:
The way you coded will work, but I think you can create those outside the loop and reuse them rather than recreate them on each loop.